src/redux/category/categorySplice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: 'Under Construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'Categories',
+  initialState,
+  reducers: {
+    checkStatus: (state, action) => ({
+      ...state,
+      categories: action.payload === 'Under construction'
+        ? 'Under construction'
+        : state.categories,
+    }),
+  },
+});
+
```

Comment:
- [ ] As requested by the previous reviewer, it is also part of the project requirement, kindly export the `checkStatus` as an action. With this you would be sure you have follow the project requirement strictly and implemented the changes requested by the previous reviewer.

Possible implementation :point_down:

```JS
export const { checkStatus } = categoriesSlice.actions
```